Tests iOS/Support/LaunchArguments.swift

```diff
@@ -12,22 +12,6 @@ struct LaunchArguments {
     let disableSentry: Bool
     let disableSnowplow: Bool
 
-    init(
-        clearKeychain: Bool,
-        clearUserDefaults: Bool,
-        clearCoreData: Bool,
-        clearImageCache: Bool,
-        disableSentry: Bool,
-        disableSnowplow: Bool
-    ) {
-        self.clearKeychain = clearKeychain
-        self.clearUserDefaults = clearUserDefaults
-        self.clearCoreData = clearCoreData
-        self.clearImageCache = clearImageCache
-        self.disableSentry = disableSentry
-        self.disableSnowplow = disableSnowplow
-    }
-
```

Comment:
Was this not being used anymore?

Reply:
Swiftlint decided that it was redundant and would fail the build.